vmdb/app/models/ems_refresh/parsers/openstack.rb

```diff
@@ -2,6 +2,7 @@
 
 module EmsRefresh::Parsers
   class Openstack < Cloud
+    include EmsRefresh::Parsers::OpenstackCommon::Images
```

Comment:
Can you change the namespace here to `EmsRefresh::Parsers::Openstack::Images`?  

We can just reuse the `Openstack` namespace in the inventory collection for shared Openstack parsing code.